tests/unit/dom.ts

```diff
@@ -237,5 +237,169 @@ registerSuite({
+							dom.containsClass(null, 'test');
+						});
+					},
+
+					'invalid class'() {
+						assert.throws(function () {
+							dom.containsClass(targetElement, 'te est');
+						});
+					},
+
+					'null class'() {
+						assert.doesNotThrow(function () {
+							dom.containsClass(targetElement, null);
+						});
+					}
+				},
+
+				removeClass: {
+					afterEach() {
+						targetElement.className = '';
+					},
+
+					'remove single class'() {
+						dom.addClass(targetElement, 'test');
+						assert.include(targetElement.className, 'test');
+						dom.removeClass(targetElement, 'test');
+						assert.notInclude(targetElement.className, 'test');
+					},
+
+					'remove multiple classes'() {
+						dom.addClass(targetElement, 'test1', 'test2');
+						assert.include(targetElement.className, 'test1');
+						assert.include(targetElement.className, 'test2');
+						dom.removeClass(targetElement, 'test1', 'test2');
+						assert.notInclude(targetElement.className, 'test1');
+						assert.notInclude(targetElement.className, 'test2');
+					},
+
+					'null node'() {
+						assert.doesNotThrow(function () {
+							dom.removeClass(null, 'test');
+						});
+					},
+
+					'invalid class'() {
+						assert.throws(function () {
+							dom.removeClass(targetElement, 'te est');
+						});
+					},
+
+					'null class'() {
+						assert.doesNotThrow(function () {
+							dom.removeClass(targetElement);
+						});
+					},
+
+					'remove nonexistent class'() {
+						assert.doesNotThrow(function () {
+							dom.removeClass(targetElement, 'random');
+						});
+					},
+
+					'remove mutiple nonexistent classes'() {
+						assert.doesNotThrow(function () {
+							dom.removeClass(targetElement, 'random', 'random1');
+						});
+					}
+				},
+
+				toggleClass: {
+					'toggling existing class removes it'() {
+						dom.addClass(targetElement, 'test');
+						assert.include(targetElement.className, 'test');
+						dom.toggleClass(targetElement, 'test');
+						assert.notInclude(targetElement.className, 'test');
+					},
+
+					'toggling nonexistent class adds it'() {
+						dom.toggleClass(targetElement, 'test');
+						assert.include(targetElement.className, 'test');
+					},
+
+					'toggling class using force = true'() {
+						dom.toggleClass(targetElement, 'test', true);
+						assert.include(targetElement.className, 'test');
+						dom.toggleClass(targetElement, 'test', true);
+						assert.include(targetElement.className, 'test');
+					},
+
+					'toggling class using force = false'() {
+						dom.toggleClass(targetElement, 'test', false);
+						assert.notInclude(targetElement.className, 'test');
+						dom.addClass(targetElement, 'test');
+						dom.toggleClass(targetElement, 'test', false);
+						assert.notInclude(targetElement.className, 'test');
+					}
+				}
+			};
+		}
+
+		return {
+			HTMLElement: cssSuite(document.createElement('div')),
+			SVGElement: cssSuite(document.createElement('svg'))
```

Comment:
I think the reason these pass even for SVG is because the SVG element is created with `document.createElement`.  In this case, at least in Chrome where I've been poking around, `className` operates like it would for any DOM node.  However, SVG elements created with `document.createElementNS` - or authored directly within an HTML document - do not behave this way, and I imagine the tests would fail for that.

So yeah, I am thinking we should punt on this for now and create NS-aware equivalents when the time comes.